### PR TITLE
fix(ISD-413): Upgrade celery-prometheus-exporter-image

### PR DESCRIPTION
**Pull request by amandahla**
This PR changes the celery-prometheus-exporter-image version to a newer one with fewer vulnerabilities.

Juju version was pinned because of this [issue](https://github.com/juju/python-libjuju/issues/863).

The integration test was changed because of the difference between the versions described [here](https://github.com/juju/python-libjuju/issues/707#issuecomment-1212296289).

### CODE REVIEW

File: metadata.yaml [hunk 1]
```diff
@@ -58,7 +58,7 @@ resources:
     type: oci-image
     description: Prometheus exporter for celery
     auto-fetch: true
-    upstream-source: danihodovic/celery-exporter:0.5.8
+    upstream-source: danihodovic/celery-exporter:0.7.6
```
Comment (jdkandersson) [inline metadata.yaml:61]: I'm not sure if there are any trivy ignores that we could potentially remove now?

Comment (amandahla) [reply metadata.yaml:61]: Only images resulting from ROCKS or from Dockerfiles are scanned by Trivy. External ones set on upstream-source in metadata file are not scanned.

**Review by jdkandersson — DISMISSED** [review]

**Review by jdkandersson — DISMISSED** [review]

**Review by merkata — DISMISSED** [review]
awesome

**Review by gregory-schiano — DISMISSED** [review]
LGTM

**Review by arturo-seijas — DISMISSED** [review]
LGTM

**Review by jdkandersson — DISMISSED** [review]

**Review by gregory-schiano — DISMISSED** [review]
LGTM

**Review by jdkandersson — APPROVED** [review]

**Review by yanksyoon — APPROVED** [review]

**Pull request merged**